src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -1,16 +1,21 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Composition;
 using System.Threading;
 using Microsoft.CodeAnalysis.Classification;
 using Microsoft.CodeAnalysis.CSharp.Extensions;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.CSharp.Classification
 {
-    internal static class ClassificationHelpers
+    [ExportLanguageService(typeof(IClassificationHelpersService), LanguageNames.CSharp), Shared]
+    internal sealed class ClassificationHelpers : IClassificationHelpersService
```

Comment:
who imports this service?

Reply:
Nobody, I forgot to revert this. Thanks!

Reply:
➡️ Fixed in fc20d2c6da76740bc23ce9ff1c012fd87fd29426